### DISCUSSION TITLE
customizing Likert scale (adding more texts)

### DISCUSSION THREAD

**Discussion by sonhkim** [jspsych/jsPsych#1522] [Q&A]
Hello! 
I currently have a Likert scale with 7 radio buttons and numbers (0-6). But I'd like to add some texts such as "Not at All", "Somewhat", "Very much" on top of the scale bar. How can I do this? 

Here's what I have now:  

var scale=["N/A", "0", "1", "2", "3", "4", "5", "6"]
var question_trial = {
      type: 'survey-likert',
      questions: function() {
      	var prompts = [
      	{
      		prompt: jsPsych.timelineVariable("question", true),
      		name: jsPsych.timelineVariable("word", true),
      		labels: scale, 
      		required: true
      	},
      	];
      	return prompts;
      },
      randomize_question_order: true,
      };

**Comment by LMD-nat** [comment]
Hi! I've been able to do this by creating a label variable with the text for each response on the likert scale like so: 
`var freq = ['Not at all', 'A little', 'Somewhat', 'About half the time', 'More than half the time', 'Most of the time', 'All the time']`

And then I refer to that variable within the likert scale code as the "labels" like this: 

````
var questionnaire = {
  type:'survey-likert',
  questions:[
    {prompt: "How often has the happened to you", name:'q_1', labels: freq, required:true}
  ],
````

Hope that helps!

**Reply by sonhkim** [reply]
Hi thanks, but I actually did the same thing already. What I want to do is to add the text _in addition to_ the number labels. I should've been clearer. I also edited the question.
